CONTRIBUTING.md

```diff
@@ -19,13 +20,13 @@ Then install the package:
 
 ```sh
 cd STEMMUS_SCOPE_Processing
-pip install -e .
+python3 -m install -e .[dev]
```

Comment:
This is not the right syntax for Windows users, at least inside a `venv` environment.

If I activate an environment, and check where python and python3 are, this is the result:
```shell
> which python3
/usr/bin/python3

> which python
~/venv/Scripts/python
```
```

Reply:
fixed.